Lombiq.Hosting.Tenants.Maintenance/Readme.md

```diff
@@ -43,14 +43,50 @@ Provides the core functionality for updating the tenants' URL based on the app c
 
 ### `Lombiq.Hosting.Tenants.Maintenance.UpdateSiteUrl`
 
-It's a maintenance task that updates the site's base URL in the site settings based on the app configuration (see `Lombiq.Hosting.Tenants.Maintenance.TenantUrlMaintenanceCore`). It is available on any tenant. To make your application execute this task, you need to add the following to your `Startup.cs`:
+It's a maintenance task that updates the site's base URL in the site settings based on the app configuration. It is available on any tenant.
+
+To make your application execute this task, you need to add the following to your `Startup.cs`:
 
 ```csharp
 public void ConfigureServices(IServiceCollection services) =>
     services.AddOrchardCms(
         builder => builder.AddTenantFeatures(Lombiq.Hosting.Tenants.Maintenance.Constants.FeatureNames.UpdateTenantUrl));
 ```
 
+The following configuration options are available to set the site URL:
+
+```json
+{
+  "OrchardCore": {
+    "Lombiq_Hosting_Tenants_Maintenance": {
+      "UpdateSiteUrl": {
+        "SiteUrl": "https://domain.com/{TenantName}"
```

Comment:
This maintenance would also require a default tenant url, because in our example on dotnest staging it has a different structure than this.